src/org/sosy_lab/java_smt/solvers/cvc5/CVC5ProofsTest.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * This file is part of JavaSMT,
+ * an API wrapper for a collection of SMT solvers:
+ * https://github.com/sosy-lab/java-smt
+ *
+ * SPDX-FileCopyrightText: 2024 Dirk Beyer <https://www.sosy-lab.org>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.sosy_lab.java_smt.solvers.cvc5;
+
+import io.github.cvc5.CVC5ApiException;
+import io.github.cvc5.Solver;
+import org.junit.AssumptionViolatedException;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Ignore;
+import org.sosy_lab.common.NativeLibraries;
+
+@SuppressWarnings({"unchecked", "rawtypes", "unused", "static-access"})
+@Ignore
+public class CVC5ProofsTest {
+  @BeforeClass
+  public static void loadCVC5() {
+    try {
+      CVC5SolverContext.loadLibrary(NativeLibraries::loadLibrary);
+    } catch (UnsatisfiedLinkError e) {
+      throw new AssumptionViolatedException("CVC5 is not available", e);
+    }
+  }
+
+  private Solver solver;
+
+  @Before
+  public void init() throws CVC5ApiException {
+    solver = createEnvironment();
+  }
+
+  private static Solver createEnvironment() throws CVC5ApiException {
+    Solver newSolver = new Solver();
```

Comment:
This tests nothing currently. Also, native proof tests are good, but should be part of the CVC5 native test-class.